python/hsfs/engine/python.py

```diff
@@ -806,6 +816,8 @@ def acked(err, msg):
 
         # make sure producer blocks and everything is delivered
         producer.flush()
+        if progress_bar is not None:
```

Comment:
I don't think we need these `is not None` checks, if it's none then it failed during initialization of the progress bar anyway.

Reply:
agree, done